sbin/newfs_msdos/newfs_msdos.8

```diff
@@ -159,6 +160,9 @@ Number of hidden sectors.
 Number of reserved sectors.
 .It Fl s Ar total
 File system size.
+.It Fl T Ar timestamp
+Specify a timestamp to be used for filesystem creation so that it can be consistent for reproducible builds.
+The timestamp can be a pathname (where the timestamps are derived from that file) or an integer value interpreted as the number of seconds from the Epoch.
```

Comment:
I think "since the Epoch" is more correct than "from the Epoch," which could also be used to mean seconds before the epoch, and since this section is already diverging from NetBSD's I think it's OK.